services/app-web/src/pages/StateSubmission/Documents/Documents.test.tsx

```diff
@@ -357,6 +385,85 @@ describe('Documents', () => {
         })
     })
 
+    describe('the delete button', () => {
```

Comment:
Hana said she’d prefer this test live in StateSubmissionForm.test.tsx instead of here since that’s the actual component under test.